requirements.txt

```diff
@@ -1,8 +1,9 @@
 CppHeaderParser>=2.7.4
 build>=1.2.2
 meson>=1.7.0
-python-magic>=0.4.27
+python-magic>=0.4.27; platform_system != "Windows"
 PyYAML==6.0.2
+setuptools==80.9.0
```

Comment:
Do we strictly need `80.9.0` or would `>=80.9.0` work as well?

Reply:
I would do >=. We don't have a strict upper feature bound we are working through. Based on my experience, I am sure we will regret either choice in a different way.

Reply:
Okay, bumped it but did not test.